config/routes.rb

```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  # root "users#index"
+  get '/users', to: 'users#index'
+  get '/users/:id(.:format)', to: 'users#show', as: 'user'
+  get './users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get './users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```

Comment:
- These routes are displaying as invalid because you have not set the routes correctly, please remove the `.` at the beginning of the routes, run `rails routes` and ensure all routes work correctly. Please fix this 😄 

Your project:
![fzdfdf](https://user-images.githubusercontent.com/73607512/236816808-9d182951-335c-4d15-8d4c-2fde5a741f24.PNG)
![rherhr](https://user-images.githubusercontent.com/73607512/236816813-8dc56a77-0691-4ead-9f2c-5d18522a0b71.PNG)


**OPTIONAL** - To help improve your application, I suggest that you have a root page for your application, This would serve as a landing page for your app. I suggest that the `user_index` path should be your root page. Please fix this 👍 👍